packages/wrangler/src/metrics/metrics-config.ts

```diff
@@ -66,7 +66,7 @@ export function getMetricsConfig({
 	const sendMetricsEnv = getWranglerSendMetricsFromEnv();
 	if (sendMetricsEnv !== undefined) {
 		return {
-			enabled: sendMetricsEnv.toLowerCase() === "true",
+			enabled: sendMetricsEnv,
```

Comment:
```suggestion
			enabled: sendMetricsEnv === true,
```

Reply:
> The type of `getWranglerSendMetricsFromEnv()` is `() => boolean | undefined`;

That's what I meant:

```ts
export const getWranglerSendMetricsFromEnv =
	getBooleanEnvironmentVariableFactory({
	variableName: "WRANGLER_SEND_METRICS",
});
```

You have to go to level higher to figure the type.

Agreed that narrowing make the `=== true` unneeded

Reply:
If you hover over the `getWranglerSendMetricsFromEnv()` function VS Code will tell you the return type is `boolean | undefined`.